tests/Functional/Action/Lti/UpdateLti1p1OutcomeActionTest.php

```diff
@@ -90,8 +93,23 @@ public function testItReturns200IfTheAuthenticationWorksAndAssignmentExists(): v
         $time = time();
         $signature = $this->generateSignature($ltiInstance, (string)$time);
 
-        /** @var string $xmlBody * */
-        $xmlBody = file_get_contents(__DIR__ . '/../../../Resources/LtiOutcome/valid_replace_result_body.xml');
+
+        $uidGenerator = $this->createMock(UuidFactoryInterface::class);
+        self::$container->set('test.uid_generator', $uidGenerator);
+
+        $uidGenerator
+            ->method('uuid4')
+            ->willReturn('e36f227c-2946-11e8-b467-0ed5f89f718b');
+
+        /** @var string $xmlRequestBody */
+        $xmlRequestBody = file_get_contents(
+            __DIR__ . '/../../../Resources/LtiOutcome/valid_replace_result_body.xml'
+        );
+
+        /** @var string $xmlResponseBody */
+        $xmlResponseBody = file_get_contents(
+            __DIR__ . '/../../../Resources/LtiOutcome/valid_replace_result_response_body.xml'
+        );
```

Comment:
Let's try to avoid using such resource files if possible. Since you don't even need a physical file here, why not just having the content as a string? Especially since you expect the UUID you defined above also to be in the response

Reply:
Fully agree - until real file is really needed for test, makes no sense to use it, IMO

Reply:
Kept it for now. Let's discuss a better approach for it.